### PR TITLE
Annotate Game Board

### PR DESCRIPTION
**Pull request by tautvydasLiekis**

### CODE REVIEW

File: services/app/src/static/style/chess-board.css [hunk 1]
```diff
@@ -1,8 +1,12 @@
 table {
-  border: 1px solid black;
   text-align: center;
 }
 
+#chess-board .white-tile,
```
Comment (tautvydasLiekis) [inline services/app/src/static/style/chess-board.css:5]: Pipeline check throws prettier error, but the code is styled accordingly to guidelines

File: services/app/src/components/chessBoard/ChessBoard.jsx [hunk 2]
```diff
@@ -11,16 +11,48 @@ const ChessBoard = ({ numberOfRows, numberOfCells }) => {
     })),
   );
 
+  const alphabetArray = [...'abcdefghijklmnopqrstuvwxyz'];
+  const numberArray = Array.from(
+    { length: board.length },
+    (_, i) => i + 1,
+  ).reverse();
+
+  board.push(
```
Comment (mykolasmolis) [inline services/app/src/components/chessBoard/ChessBoard.jsx:20]: Please fill `xAxis` attribute using the `.map` above.

File: services/app/src/components/chessBoard/ChessBoard.jsx [hunk 3]
```diff
@@ -11,16 +11,48 @@ const ChessBoard = ({ numberOfRows, numberOfCells }) => {
...
+                  : 'xAxis' in tile
+                  ? tile.xAxis
+                  : 'yAxis' in tile
+                  ? tile.yAxis
+                  : ''
+              }
               key={tileIndex}
-            />
+            >
+              {'yAxis' in tile ? tile.yAxis : tile.xAxis}
```
Comment (mykolasmolis) [inline services/app/src/components/chessBoard/ChessBoard.jsx:54]: `tile.yAxis || tile.xAxis`

File: services/app/src/components/chessBoard/ChessBoard.jsx [hunk 4]
```diff
@@ -11,16 +11,48 @@ const ChessBoard = ({ numberOfRows, numberOfCells }) => {
...
+    (_, i) => i + 1,
+  ).reverse();
+
+  board.push(
+    [...Array(numberOfCells).keys()].map((_tile, tileIndex) => ({
+      xAxis: alphabetArray[tileIndex],
+    })),
+  );
+
+  board.forEach((element, i) => {
```
Comment (mykolasmolis) [inline services/app/src/components/chessBoard/ChessBoard.jsx:26]: Same as above

Comment (mykolasmolis) [reply services/app/src/components/chessBoard/ChessBoard.jsx:26]: It looks like you want to add Annotations as additional column and row. This logic should be part of the logic in line 6. Let's catch up to see what it looks like.

**Review by mykolasmolis — APPROVED** [review]

**Pull request merged**